subprojects/docs/README.md

```diff
@@ -145,11 +145,71 @@ Let's break down this example to explain:
 
 It is possible to embed sample sources, commands, and expected output directly in the Asciidoc (or a mixture of embedded and `include`d), but we don't use this for the user manual yet. See the [Exemplar documentation](https://github.com/gradle/exemplar/#configuring-embedded-samples) if you're interested in this.
 
-### Testing samples and snippets
+## Testing docs 
 
-#### Code samples
+Currently, `docs` is tested by `docs:docsTest`, which covers three kinds of code: 
 
-Samples and output belong under `src/samples` and are published beside the user manual. See the `org.gradle.samples` plugin.
+- The code generated by [Build Init Plugin](https://docs.gradle.org/current/userguide/build_init_plugin.html), i.e. `gradle init` task.
+  - `generate-samples.gradle.kts` registers multiple generator tasks that generate same sample project code as you manually run `gradle init`.
```

Comment:
The samples generated using `init` are also published with the user manual. I think this should be mentioned because with the current text it looks like they are not.

Reply:
Added.